config/locales/references.yml

```diff
@@ -0,0 +1,894 @@
+---
```

Comment:
so we discussed the need for officers to amend this will come very soon - how can we have yaml policies if we want these to be editable by officers? The policies change every year, are we going to create a dependency on developers having to maintain a 900 line YAML file instead of seeding these into the DB and having them be editable?

I might not understand enough though - is there a way for us to build an interface to edit things that are put in using yaml? I really, really don't want to spend hours of "development" amending text so just making sure we aren't creating that burden for future maintainers...

Reply:
YAML is best for that sort of verbose copy, at least for the time being since we don't want to build a fully-fledged system to copy the legislation. What alternatives are you thinking of?

Reply:
What about storing them in the database but have a nightly job to pull the latest changes from the API? That way you avoid having to build an interface.

Reply:
not keen on storing all this copy in the DB + Tom G wanted to "try out" this feature and see if it works with officers so I'd rather we stick to this plain solution now and worry about how to take it next-level once it's proven.

Reply:
I'll check with the product owners whether they're happy with the exact copy of the API (well, the XML resources). I seem to remember they wanted to use a slightly simpler copy but I'll double check.

Reply:
could you help me understand why the YAML is better than adding text to the database so it can be edited later? I was trying to find some articles on YAML rendering vs retrieving the data from a db but can't find anything good, would really like to understand the logic behind that decision 🙏

Reply:
YAML is incredibly easy to read and maintain, it's also excellent for text-heavy, verbose content which is exactly what this is. I assume that's why Rails defaults to a bunch of YAML files in `config/locales/` for its translations. Those legal texts are not data, they don't need to be queried or joined (not at this point anyway) so they don't belong in the database. Building an admin interface might be as easy as dropping a gem (`activeadmin` springs to mind) but I don't want to introduce that complexity for a feature that we are only trialling at the moment. YAML just works, and works great for this usecase I think.

If the feature is approved, and the update process turns out to be a pain we can move things in the DB. This is good enough a start.

Reply:
gotcha, database entries should be for stuff we want to query and join - thanks for explaining! 
If they decide they do want this feature in the long run its likely they'll want us to display which policy classes were approved and have those display on the decision notice - but we'll cross that bridge when we come to it